### DISCUSSION TITLE
SOC-CUSTOMIZATION to DISABLE RULE

### DISCUSSION THREAD

**Discussion by leber2** [Security-Onion-Solutions/securityonion#8855] [Unsupported Versions]
Hi,
 
I'm trying to incorporate this [https://docs.securityonion.net/en/2.3/managing-alerts.html#disable-the-sid](url) through SOC-customization. Was wondering if there is an option to disable rule in the WebUI, like clicking on the entry and having the option to disable, etc.

Any suggestion would be highly appreciated.
Thanks.

**Comment by reyesj2** [comment]
Not currently, but keep an eye out for [Security Onion 2.4](https://blog.securityonion.net/2022/10/sneak-peek-at-security-onion-24.html)

I would add that currently there is a neat wrapper script for your use in disabling / enabling / modifying rules. try it out

```
sudo so-rule
```
